src/loaders/HandCashItemsLoader.ts

```diff
@@ -57,6 +57,11 @@ export class HandCashItemsLoader extends AbstractItemsLoader {
         };
     }
 
+    private async getContentType(url: string): Promise<string> {
```

Comment:
yes - I will include in the next iteration because it will also include some breaking changes to some of the input types 

Same time we deprecate imageHighResUrl I think it makes sense